network_simulator/link.py

```diff
@@ -50,6 +104,21 @@ def __eq__(self, other):
         return isinstance(other, self.__class__) and \
             self.__dict__ == other.__dict__
 
+    def get_packet(self):
```

Comment:
They aren't being used, just wanted to put them in in case we wanted to them at some point. Took out for now